### PR TITLE
[PositionerSetup] add option "Checking ONID/TSID"

### PR DESCRIPTION
**Pull request by Dima73**
-"Checking ONID/TSID" use only manual.
Enter valid onid/tsid and run check.
-add option "Stop live TV service"
-update last sat rotor position
-small cosmetics code

### CODE REVIEW

File: lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py [hunk 1]
```diff
@@ -778,17 +786,48 @@ def tune(self, transponder):
...
+			self.onid = tsidonid[0]
+			self.tsid = tsidonid[1]
+			if self.frontend and self.isLocked() and not self.isMoving and hasattr(self, "raw_channel") and self.raw_channel:
+				self.checkingTsidOnid = True
+				self.raw_channel.receivedTsidOnid.get().append(self.gotTsidOnid)
+				self.raw_channel.requestTsidOnid()
+
+	def gotTsidOnid(self, tsid, onid):
+		if tsid == self.tsid and onid == self.onid:
+			msg = "\c0000??00" + _("This valid ONID/TSID")
```
Comment (nautilus7) [inline lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: Why do you keep hardcoding colors in the code?? Do you understand that skins cannot change these colors? This is unacceptable!

Comment (Dima73) [reply lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: `This is unacceptable!`
Why?
Green color ok.
Red color not ok.


What else does the user just need?
I think in 10 seconds of displaying the message, the user will tolerate these colors.

Comment (betacentauri) [reply lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: Eg some suffer red green weakness and can’t see the difference.
But I don’t think nautilus7 means this.

Comment (nautilus7) [reply lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: @Dima73 the purpose of skins is to define the colors THEY want (among other things). You do not allow them to do so with your coding....

A few years ago, I tried to clear all hard coded colors, but I see them coming back again. And you did it one more time recently with another commit (that I can't find right now).

Comment (nautilus7) [reply lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: If you think that adding colors gives some value to your code, then this is how I propose to do it https://github.com/OpenPLi/enigma2/commit/17f344a92cb05354e26179227b6ee49703d3c7e9

Comment (Dima73) [reply lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: Let's apply this patch.
Will you create a new one with custom colors later? 
Ok?

Comment (nautilus7) [reply lib/python/Plugins/SystemPlugins/PositionerSetup/plugin.py]: I am not in a mood of fixing other people's incomplete/scrappy code....

Btw, what does this patch do? Your description is not detailed at all.

**Pull request merged**